### PR TITLE
add password field to MPDClient()

### PR DESCRIPTION
**Pull request by chebro**
Changes:
- Included a "pass" field in the default config.
- Added `MPDClient().password()` for password protected clients.

### CODE REVIEW

**Comment by chebro** [comment]
This might seem like a useless feature, but I happen to host my mpd server on my raspberry pi(which I alone want to control), so I use a password :D

File: config.example [hunk 1]
```diff
@@ -7,3 +7,4 @@ image_method = pixcat
 [mpd]
 host = localhost
 port = 6600
+pass = example
```
Comment (GuardKenzie) [inline config.example:10]: This line should be commented out (`# pass = example`) to reflect that the default value for `pass` is no value at all.

Comment (GuardKenzie) [inline config.example:10]: This line should be commented out (`# pass = example`) to reflect that the default value for `pass` is no value at all.

**Review by GuardKenzie — CHANGES_REQUESTED** [review]
This is some good work! Just that one issue with the config.

File: config.example (hunk 1 above)
Comment (chebro) [reply config.example]: whoops, that didn't cross me. fixed it now :)

**Pull request merged**